packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -742,12 +742,30 @@ module.exports = (log, db, mailer, Password, config, customs, subhub, signinUtil
           query: {
             uid: isA.string().min(32).max(32).regex(HEX_STRING)
           }
+        },
+        response: {
+          schema: {
+            authenticationMethods: isA.array().items(isA.string().required()).optional(),
+            authenticatorAssuranceLevel: isA.number().min(0).optional(),
+            exists: isA.boolean().required(),
+            locale: isA.string().optional(),
+          }
         }
       },
       handler: async function (request) {
         const sessionToken = request.auth.credentials;
         if (sessionToken) {
-          return { exists: true, locale: sessionToken.locale };
+          const account = await db.account(sessionToken.uid);
+          const amrValues = await authMethods.availableAuthenticationMethods(db, account);
+          const authenticationMethods = Array.from(amrValues);
+          const authenticatorAssuranceLevel = authMethods.maximumAssuranceLevel(amrValues);
```

Comment:
I would keep the names `availableAuthenticationMethods` and `maximumAssuranceLevel` rather than changing them. They're deliberately named differently to the corresponding properties in the sessionToken because they mean different things:

* `availableAuthenticationMethods` is a property of the account and tells you all the possible authentication methods that *might* be used with this account; `authenticationMethods` is a property of a particular sessionToken and tells you which methods *were actually used* to produce that token.
* `maximumAssuranceLevel` is a property of the account and tells you the *maximum possible* assurance level that a token for this account could have; `authenticatorAssuranceLevel` is a property of a particular sessionToken and tells you the level that *was actually achieved* by the holder of that token.

Reply:
That I can't point you to an existing doc with those words in it, is a failure of documentation when we initially landed this feature, sorry :-(

Reply:
the name is from https://github.com/mozilla/fxa/blob/17088bdae2ee97fd72a8892194cd16a7084356e9/packages/fxa-auth-server/lib/routes/account.js#L844 

Should we have 2 difference ones?

Reply:
Hrm, OK sorry, apparently I didn't follow my own advice in a previous iteration of this :-(

Consistency seems more valuable here so I'm OK with keeping what you have. But I also wonder, could you just call that existing `/profile` endpoint to get the info you need?

Reply:
> But I also wonder, could you just call that existing /profile endpoint to get the info you need?

/status made more sense to use there, but I think profile will suffice.